### DISCUSSION TITLE
.CSV extension not working

### DISCUSSION THREAD

**Discussion by MaelVe** [MediatedCommunications/Mime-Detective#13] [Q&A] [answered]
Hello, first of all thank you for your work !

Is there a validation for csv file ? it's not working on my side.

Thank you !

**Comment by TonyValenti** [comment]
@MaelVe Which definition set are you using?

**Reply by MaelVe** [reply]
i'm using : 
`var Inspector = new ContentInspectorBuilder() {
    Definitions = Definitions.Default.All()
}.Build();`

i tried the other ones and i got an error for each (not finding the using)

**Reply by mo2eharrington** [reply]
Hi, was this ever resolved?  I am using ExhaustiveBuilder and Inspector.Inspect returns a null.  It's working great for all other types I can fine except for CSV.  Thanks!

**Reply by TonyValenti** [reply, marked as answer]
Hi All,
Because there is no definitive pattern or identifier to CSV files, you will likely not be able to reliably detect them from a raw stream.

**Comment by mo2eharrington** [comment]
Ah, makes sense.  Thanks!
________________________________
